test/test1000/proof.answer

```diff
@@ -1 +1 @@
-51a59b17d1858b5503cbb31fc6e3cae97d556c82d02e1978bcf276d07061fa83  test/test1000/proof
+75abf21da603bb2f56932655d2111d646aa17355b817453aedd64082ead04bd7  test/test1000/proof
```

Comment:
This is no longer valid checksum after your last fixup commits.

Perhaps just squash `create_enum: Add require to check for EOF` and checksum update into the main commit?

Reply:
Updated hashes and squashed into single commit.